src/rocksdb.rs

```diff
@@ -2106,15 +2110,49 @@ pub fn set_external_sst_file_global_seq_no(
     }
 }
 
+pub fn load_latest_options(
+    dbpath: &str,
+    env: &Env,
+    ignore_unknown_options: bool,
+) -> Result<Option<(DBOptions, Vec<CColumnFamilyDescriptor>)>, String> {
+    const ERR_CONVERT_PATH: &str = "Failed to convert path to CString when load latest options";
+
+    let dbpath = CString::new(dbpath.as_bytes()).map_err(|_| ERR_CONVERT_PATH.to_owned())?;
+    let db_options = DBOptions::new();
+    unsafe {
+        let mut raw_cf_descs: *mut *mut crocksdb_ffi::ColumnFamilyDescriptor = ptr::null_mut();
+        let mut cf_descs_len: size_t = 0;
+
+        let ok = ffi_try!(crocksdb_load_latest_options(
+                              dbpath.as_ptr(),
+                              env.inner,
+                              db_options.inner,
+                              &mut raw_cf_descs,
+                              &mut cf_descs_len,
+                              ignore_unknown_options));
+        if !ok {
+            return Ok(None)
+        }
+        let cf_descs_list = slice::from_raw_parts(raw_cf_descs, cf_descs_len);
+        let cf_descs = cf_descs_list.into_iter().map(|raw_cf_desc| {
+            CColumnFamilyDescriptor::from_raw(*raw_cf_desc)
```

Comment:
I think it should be `raw_cf_desc` instead of `*raw_cf_desc`?

Reply:
I worry about the memory manage here - -.

Reply:
I have run a test with valgrind, there is no memory leak